tests/unit/amo/sagas/test_collections.js

```diff
@@ -763,6 +764,14 @@ describe(__filename, () => {
 
       const fetchAction = await sagaTester.waitFor(expectedFetchAction.type);
       expect(fetchAction).toEqual(expectedFetchAction);
+
+      const expectedRemovedAction = addonRemovedFromCollection();
+
+      const removedAction = await sagaTester.waitFor(
```

Comment:
I would rather see this moved to its own test. In the case of breakage, it will be easier to read the test output and know how many things broke.